### DISCUSSION TITLE
Custom Endpoints; Component href option potentially related?

### DISCUSSION THREAD

**Discussion by Jaiieq** [livewire/flux#217] [Feature Requests]
Would it be possible to get custom endpoints just like Livewire? To achieve it now I have to modify the AssetManager.php file, which of course will get overwritten every time it is updated.

Also, I'm not sure if this is related to the endpoints but when using the HREF option on the components that support it I am having to manually put in the link.  If I use a route name the link is incorrect.

Example link:     domain.com/sitename/profile
Example route:  Route::get('profile', [ProfileController::class, 'show')->name('profile.show')

- Giving it a href of 'profile' produces the example above.
- Using a href of {{ route('profile.show') }} produces: domain.com/profile

**Comment by joshhanley** [comment]
@Jaiieq thanks for the request, there is already a request open for custom routes, see discussion #171, so I'm going to close this one.
